dojo/1.11/modules.d.ts

```diff
@@ -731,3 +731,14 @@ declare module 'dojo/window' {
 	const window: dojo.WindowModule;
 	export = window;
 }
+
+declare module 'dojo/Stateful' {
```

Comment:
I added this to fix another issue just now in a6b105a1a738b596eab3d3ebcd937c2f41072ef1